src/components/login-form.yml

```diff
@@ -8,6 +8,9 @@ loginForm:
   forgotPassword:
     en: Forgot your password?
     de: Kennwort vergessen?
+  registrationLink:
+    en: No account? Sign up
+    de: Kein Konto? Registrieren
```

Comment:
I realized that in German the word "registrieren" is not really nice. And in English the difference between "sign up" and "sign in" is also confusing less tech-aware users. We could use the word "sign" actually only for the actual existing users' login, and for not-yet-existing users use the wording "create account":

* "No account? Create one"
* "Kein Konto? Erstelle eines"

(... and maybe even rename the `register` page to `create-account` at some time)

Whatcha think?

Reply:
I like that! 👏 
I'll change the translations. For now I'll not touch the page name to not escalate this too much, but it would be good if we did that too afterwards.

Reply:
Nice :clap: